engine/standalone/app/src/universal/bin/nussknacker-standalone-entrypoint.sh

```diff
@@ -2,6 +2,7 @@
 
 NUSSKNACKER_DIR=`dirname "$0" | xargs -I{} readlink -f {}/..`
 export STORAGE_DIR="$NUSSKNACKER_DIR/storage"
+PROCESSES_DIR="$NUSSKNACKER_DIR/processes"
```

Comment:
IMO process should be nested in storage. It is the category of files that you would like to keep on some external volume and be sure that after container recreation it won't disappear